src/client/reducers/reducer_movies.js

```diff
@@ -0,0 +1,12 @@
+import C from '../actions/constants';
+
+export default function (state = null, action) {
+    switch (action.type) {
+        case C.POPULATE_MOVIES:
+            return action.payload;
```

Comment:
@vitkuz normally when we populate movies it's being stored in the store. but using netfilix roulette it's common to get completely different list of movies every time, let's do it like this: 
 if there are no movies for this author or title in the store, send a new request. If there are some, they are taken from the redux store.

Reply:
I think with new APIs it is not relevent / or I don't understand